05/app.js

```diff
@@ -10,34 +10,33 @@ if (curr) {
     sectionElement.appendChild(newButton);
     console.log(newButton);
 
-// dla wszystkich elementów, które są rodzeństwem dla elementu o klasie .js-curr dodaj klasę .siblings (wykorzystaj pętlę)
+    // dla wszystkich elementów, które są rodzeństwem dla elementu o klasie .js-curr dodaj klasę .siblings (wykorzystaj pętlę)
 
-const jsParent = document.querySelectorAll('.articles__item');
-const allArticle = [];
+    const allElements = document.querySelector('.articles__item')
```

Comment:
Skoro wiele to powinno być `document.querySelectorAll()`
Pamiętaj, aby wykluczyć element o klasie `js-curr`